pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -72,7 +72,13 @@ func (c *Controller) Create(b *v3.EtcdBackup) (runtime.Object, error) {
 		return b, err
 	}
 	if cluster.Spec.RancherKubernetesEngineConfig.Services.Etcd.BackupConfig == nil {
-		return b, fmt.Errorf("[etcd-backup] cluster doesn't have a backup config")
+		v3.BackupConditionCreated.False(b)
```

Comment:
@moelsayed we should disable backup action on the cluster having nil backup config, on the api layer. Conditions are mostly used for the backend failures